<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>823</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushBtnSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>Open Test Files</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 7px;
font: 16pt &quot;Ink Free&quot;;
color: rgb(0, 0, 255);
border-color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
border-width:2px;</string>
    </property>
    <property name="text">
     <string>Select Tests</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnRun">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>710</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>34</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>Run Selected Tests</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(0, 255, 0);
color:&quot;[enabled=\&quot;true\&quot;]&quot;&quot;{ color: grey }&quot;&quot;[enabled=\&quot;false\&quot;]&quot;&quot;{ color: black }&quot;;
border-radius: 7px;
font: 34pt &quot;Ink Free&quot;;
border-width:2px;
border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnRefreshLogs">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>700</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>Upload Test-Logs</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 7px;
font: 16pt &quot;Ink Free&quot;;
color: rgb(0, 0, 255);
border-color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
border-width:2px;</string>
    </property>
    <property name="text">
     <string>Refresh Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnStop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>710</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>34</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(255, 0, 0);
border-radius: 7px;
font: 34pt &quot;Ink Free&quot;;
border-width:2px;
border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 4px;
border-width:2px;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
font: 8pt &quot;Ink Free&quot;;</string>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushBtnSelectAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 7px;
font: 12pt &quot;Ink Free&quot;;
border-color: rgb(0, 0, 255);
border-width:2px;
color: rgb(0, 0, 0);
color:&quot;[enabled=\&quot;true\&quot;]&quot;&quot;{ color: gray }&quot;&quot;[enabled=\&quot;false\&quot;]&quot;&quot;{ color: black }&quot;</string>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnUnselectAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 7px;
font: 12pt &quot;Ink Free&quot;;
border-color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
border-width:2px;
background-color:&quot;[enabled=\&quot;false\&quot;]&quot;&quot;{ color: gray }&quot;&quot;[enabled=\&quot;true\&quot;]&quot;&quot;{ color: black }&quot;</string>
    </property>
    <property name="text">
     <string>Unselect All</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>49</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Test-Runner</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
font: 10pt &quot;Nirmala UI Semilight&quot;;
border-radius: 4px;
border-width:2px;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
